lang/main-de.json

```diff
@@ -11,6 +11,7 @@
         "disabled": "Sie können keine Teilnehmer einladen.",
```

Comment:
Nur wenn du es weißt: Fehlen diese Übersetzungen in der Offiziellen Version von Jitsi-Meet?